.goreleaser.develop.yaml

```diff
@@ -10,8 +10,6 @@ env:
 
 before:
   hooks:
-    - go mod tidy
-    - make operator-ui
```

Comment:
We no longer need to make the op assets since they are generated and committed already.

Tidy should not be necessary, for the same reason - we already check that it is up to date along with generation. (I think we would prefer to fail here too, rather than silently cleaning things up that are in a bad state, and upgrading dependencies without review).